### DISCUSSION TITLE
Old Mac and old version of OS

### DISCUSSION THREAD

**Discussion by OldSubSailor** [Homebrew/discussions#160] [Everyday usage]
I am trying to help a fellow on on the  Ortho4xp install for Macusers X-Plane forum.  I think that he really got himself in a hole before he asked for help. 
The old Ortho4 install instructions read in part:
1) In a Terminal window, install Homebrew (brew.sh), a package manager
/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
I think that he and the other folk are up to the bash install part now,
then the rest of the instructions are in part (the developer created this in a unix shell script) and the majority of the folks who come to that help topic for the most part have never heard of the terminal. 
# Install dependencies
brew install python gdal spatialindex p7zip
==================================================
He is running an old MacBook under  Macos 10.12.6 and further says "i cant upgrade my Macbook is too old."

I do not know what he did but it looks like he managed to look up a way to uninstall homebrew or brew packages  etc and python packages and dig the hole.  
The above brew install python step is death to his computer, and regardless of my telling him not to do that he has done it over and over. 

That is the history at this point.  He keeps sending me long lines of brew error codes and I just need some clarification.

Can he, with his 10.12 os install the basic homebrew package to start with if I can just get him to slow down and stop at that point and after I can hopefully get him to get rid of python 3.9 which I think is also fatal to that old OS?

**Comment by SMillerDev** [comment]
Any package would try (and probably fail) to build from source because we don't support anything below 10.14 anymore. I don't think Python 3.9 would be any worse than other python versions though.

**Reply by OldSubSailor** [reply]
Alas, so no legacy? Let me pass that on to him.

**Comment by OldSubSailor** [comment]
Thank you for the reply,
Alas, there are some unfortunate misfires with Python3.9 and pip3 installs.

**Comment by OldSubSailor** [comment]
I got an email in the X-Plane forum and I hate to waste band width, I am pasting the below. 
Does this look like homebrew is installed and somewhat functional on his OS, but again killed by python?

**Comment by tresf** [comment]
I've used [Tigerbrew](https://github.com/mistydemeo/tigerbrew) with limited success on some old machines.  When they're so old that OpenSSL fetches fail, it's pretty hard to recover.  I wouldn't expect 10.12 to suffer this problem specifically though.

Although the user states that they can't upgrade their OS the environment has two unsupported paths:
* Use Homebrew on an unsupported OS (or Tigerbrew)
* Use a Homewbrew-supported OS on unsupported hardware by force installing a supported operating system on older hardware.

I've successfully upgraded a very old mac mini to 10.14 using [dosdude1's](http://dosdude1.com/software.html) patched installers.  Although they come with their own risks, they're widely used by Mac users and were specifically intended for the legacy use-case.  As always, use at your own risk. :)
